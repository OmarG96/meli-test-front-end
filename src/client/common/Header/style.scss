@import "../../utils/scss/variables.scss";

header {
  height: 52px;
  background-color: $yellow;
  padding: 0 5%;
  display: grid;
  grid-template-columns: repeat(12, auto);
  grid-gap: 16px;
}

.header-content {
  grid-column: 2 / span 10;
  display: flex;
  align-items: center;
  justify-content: center;
}
