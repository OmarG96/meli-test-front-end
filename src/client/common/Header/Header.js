import React from "react";
import { node } from "prop-types";

import "./style.scss";

const Header = ({ children }) => {
  return (
    <header>
      <div className="header-content">{children}</div>
    </header>
  );
};

Header.propTypes = {
  children: node,
};

export default Header;
