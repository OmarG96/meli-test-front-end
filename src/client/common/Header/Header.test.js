import React from "react";
import Header from ".";
import { fireEvent, render } from "@testing-library/react";

describe("GIVEN SearchBar", () => {
  describe("WHEN is rendered", () => {
    it("THEN match to snapshot", () => {
      const { asFragment } = render(
        <Header>
          <div>example</div>
        </Header>
      );
      expect(asFragment()).toMatchSnapshot();
    });
  });
});
