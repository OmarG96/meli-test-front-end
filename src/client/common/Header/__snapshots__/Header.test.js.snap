// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GIVEN SearchBar WHEN is rendered THEN match to snapshot 1`] = `
<DocumentFragment>
  <header>
    <div
      class="header-content"
    >
      <div>
        example
      </div>
    </div>
  </header>
</DocumentFragment>
`;
